dodola/tests/test_services.py

```diff
@@ -582,48 +582,22 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     )
 
     # remove leap days
-    temp_slice = convert_calendar(ds["air"], target="noleap")
+    ref_fine = convert_calendar(ref_fine["temperature"], target="noleap")
 
     # take the mean across space to represent coarse reference data for AFs
-    temp_slice_mean = temp_slice.mean(["lat", "lon"])
-    # then tile it to be on the same grid as the fine reference data
-    temp_slice_mean_resampled = temp_slice_mean.broadcast_like(temp_slice)
-
-    # need to create some fake bias corrected data so that we can use it to downscale
-    with set_options(sdba_extra_output=True):
```

Comment:
Since we're removing these, could we ditch the `from xclim import sdba, set_options` line at the top of this file?
Same for `from xclim.sdba.utils import equally_spaced_nodes` and `from numpy.testing import assert_approx_equal`. I don't think they're used anymore.

Reply:
Yep - thanks for noticing!